infra/gcp/create_bootstrap_vm.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
I see this is in parallel to Azure, but we don't have a markdown file describing steps that use this VM. Do you now have a series of steps for using terraform in GCP?

Reply:
There is the `infra/gcp/README.md` which details the instructions of using terraform in GCP, but I will update it with a reference to this script.